### PR TITLE
Use primal to sync nostr contacts

### PR DESCRIPTION
**Pull request by benthecarman**
Getting some errors with https://github.com/MutinyWallet/mutiny-web/pull/412 but I don't think it is my fault

### CODE REVIEW

**Comment by TonyGiorgio** [comment]
I'll wait until you & @futurepaul resolve the errors.

File: mutiny-core/src/lib.rs [hunk 1]
```diff
@@ -404,22 +405,31 @@ impl<S: MutinyStorage> MutinyWallet<S> {
...
-        #[cfg(target_arch = "wasm32")]
-        client.add_relay("wss://relay.damus.io").await?;
-
-        #[cfg(not(target_arch = "wasm32"))]
-        client.add_relay("wss://relay.damus.io", None).await?;
-
-        client.connect().await;
+        let body = json!(["contact_list", {"pubkey": npub, "extended_response": true }]);
+
+        let url = primal_url.unwrap_or("https://dev.primal.net/api");
```
Comment (TonyGiorgio) [inline mutiny-core/src/lib.rs:413]: Needs to point to our own.

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]

File: mutiny-core/src/lib.rs (hunk 1 above)
Comment (benthecarman) [reply mutiny-core/src/lib.rs]: Fixed, it is still syncing events so might be a bit

**Comment by TonyGiorgio** [comment]
"acess to fetch at 'https://primal-cache.mutinywallet.com/api' from origin 'http://localhost:3420' has been blocked by CORS policy: Re"

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**